patches/server/0738-Expose-tracked-entities.patch

```diff
@@ -0,0 +1,41 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: SuperKooks <superkooks@superkooks.com>
+Date: Wed, 4 Aug 2021 18:05:01 +1000
+Subject: [PATCH] Expose tracked entities
+
+
+diff --git a/src/main/java/net/minecraft/server/level/ChunkMap.java b/src/main/java/net/minecraft/server/level/ChunkMap.java
+index 42fd259f4492e539112b5bcb310aaaadab58a443..eb412fac2e4ad7e292eaf9fee39acdfd87c2d775 100644
+--- a/src/main/java/net/minecraft/server/level/ChunkMap.java
++++ b/src/main/java/net/minecraft/server/level/ChunkMap.java
+@@ -2279,7 +2279,7 @@ Sections go from 0..16. Now whenever a section is not empty, it can potentially
+ 
+         final ServerEntity serverEntity;
+         final Entity entity;
+-        private final int range;
++        public final int range; // Paper - expose range to entities
```

Comment:
Unsure about this, but don't we use `build-data/paper.at` for this instead or manually exposing the field in a patch ?

Reply:
You're probably right. The CONTRIBUTING.md didn't mention build-data. How would I go about doing this?

Reply:
Checkout the previous mentioned file, it contains a neat block of documentation at the top. Add your change as the bottom to expose the field including a comment (you'll see what others did before you XD). After applying patches, the field should be public in your local setup.

Reply:
Thanks!

Reply:
Does this mean the build should error?

Reply:
@kennytv ?